.github/workflows/publish-python.yml

```diff
@@ -82,5 +82,5 @@ jobs:
     - name: Publish layer
       env:
         LAYER_NAME: "coralogix-opentelemetry-python-wrapper"
-        COMPATIBLE_RUNTIMES: "python3.8 python3.9 python3.10 python3.11 python3.12"
```

Comment:
Please remember to do the same in telemetry exporter

Reply:
Yes. I just created the PR.

Thanks!